PolyEngine/Engine/Src/Input/InputWorldComponent.hpp

```diff
@@ -65,6 +70,8 @@ namespace Poly
 		std::unordered_map<size_t, ControllerState> Controllers;
 		Dynarray<Optional<size_t>> PlayerIDToJoystickID;
 		std::unordered_map<size_t, size_t> JoystickIDToPlayerID;
+		bool IsConsumed;
+		char* CharUTF8;
```

Comment:
default values for both of those fields are needed.

Reply:
Done, please check it